src/plugins/presentation_util/public/components/dashboard_drilldown_options/types.ts

```diff
@@ -12,10 +12,14 @@ export type DashboardDrilldownOptions = {
   useCurrentFilters: boolean;
   useCurrentDateRange: boolean;
   openInNewTab: boolean;
+  overflowTextWrap: boolean;
+  overflowEllipsis: boolean;
```

Comment:
Also, I think this should be a setting **for the entire links panel**, not per-link. What do you think @rshen91?

Reply:
@andreadelrio can be our tie breaker, hahaha :)

Reply:
I think this should be a setting for the entire links panel.